### ISSUE TITLE
Xonomy.harvest() adds additional space attribute

### ISSUE THREAD

**Issue by chippyash** [michmech/xonomy#40]
The harvest() method is adding additional attributes into the document: I see 
`xml:space='preserve'` being added to every element.  I can strip it out on receiving at the server but why is there in the first place?

**Comment by rtosman** [comment]
I agree, this should not be added, it actually breaks the document for the tool I am feeding the output to. I have deleted the line (around line 90) in xonomy.js in my local copy.
